deployment/staging/deployment.yaml

```diff
@@ -38,3 +38,8 @@ spec:
               value: wss://bot-busters.staging.kryha.dev
             - name: PLAYERS_PER_MATCH
               value: "4"
+            - name: NEXT_PUBLIC_RECAPTCHA_SITE_KEY
```

Comment:
You don't need to get the public key from the secrets.
For now it's ok to paste the actual key here as value. (It's public anyway). 
In the future we can consume it from the env vars declared in Github

Reply:
You should use this approach for the `RECAPTCHA_SECRET_KEY`. I don't see that anywhere.